src/sys/windows/tcp.rs

```diff
@@ -494,6 +518,9 @@ impl ListenerImp {
             State::Empty => {}
             _ => return
         }
+
+        me.iocp.unset_readiness(EventSet::readable());
```

Comment:
I think this may want to be deferred until the actual `mem::forget` below. If the `me.iocp.defer` branch is taken the readiness should still be applicable.